packages/@cdktf/cli-core/src/lib/synth-stack.ts

```diff
@@ -243,3 +258,19 @@ Command output on stdout:
     await sendTelemetry("synth", { error: true, synthOrigin });
   }
 }
+
+function getRelativeTerraformModules() {
+  let cfg;
+
+  try {
+    cfg = CdktfConfig.read();
+  } catch {
```

Comment:
We should at least add a trace log with the error in order to be able to debug this